src/Decision/BucketingManager.php

```diff
@@ -25,6 +25,9 @@ public function __construct(HttpClientInterface $httpClient, BucketingConfig $co
 
     protected function sendContext(VisitorAbstract $visitor)
     {
+        if (count($visitor->getContext())<=4){
```

Comment:
4 ?

Reply:
ok in that case, can you create a method `getDefaultContextKeys()` which return an array of the context keys and in that function use `if (count($visitor->getContext())<= count(getDefaultContextKeys())){`

or if you can't, simply add a contant `NB_MIN_CONTEXT_KEYS = 4`.